src/libDirectoryService/FinalBlockPreProcessing.cpp

```diff
@@ -177,7 +177,7 @@ void DirectoryService::ComposeFinalBlockCore()
                       blockNum, timestamp, microblockTrieRoot, stateRoot,
                       numTxs, numMicroBlocks, m_mediator.m_selfKey.second,
                       lastDSBlockNum, dsBlockHeader, m_viewChangeCounter),
-        isMicroBlockEmpty, microBlockTxHashes,
+        vector<bool>(isMicroBlockEmpty), vector<TxnHash>(microBlockTxHashes),
```

Comment:
Just found another place that could use {} initialization.

Reply:
Will update in a later pr